hardware/raspberrypi/revision-codes/README.md

```diff
@@ -0,0 +1,94 @@
+# Raspberry Pi revision codes
+
+Each distinct Raspberry Pi model revision has a unique revision code. You can look up a Raspberry Pi's revision code by running:
+
+```bash
+cat /proc/cpuinfo
+```
+
+The last three lines show the hardware type, the revision code and the Pi's unique serial number, for example:
+
+```
+Hardware    : BCM2835
+Revision    : a02082
+Serial      : 00000000765fc593
+```
+
```

Comment:
Worth checking RPi.GPIO - I think it might use that for detecting which offset into /dev/mem to use when /dev/gpiomem isn't available. The reason I mention that is that the fallback "native" implementation in GPIO Zero certainly needs updating (it does rely on the Hardware line) and I'm fairly sure I got that logic from RPi.GPIO...